src/renderer/components/watch-video-playlist/watch-video-playlist.vue

```diff
@@ -6,9 +6,12 @@
     <div
       v-else
     >
-      <h3>
+      <h3
+        class="playlistTitle"
+        :title="`${playlistTitle}`"
```

Comment:
You don't have to create a string from a string, the code can be simplified to this:
```suggestion
        :title="playlistTitle"
```